src/Chapter2.hs

```diff
@@ -336,8 +348,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
-
+subList a b x
+  | a<0 || b <0 = []
+  | otherwise = take (b-a+1) (drop a x)
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you can implement it more efficiently by adding a separate case to your check above :slightly_smiling_face: